spec/truffle/interop/keys_spec.rb

```diff
@@ -27,5 +27,19 @@ def initialize
   it "returns the instance variables of something that isn't a hash" do
     Truffle::Interop.keys(InteropKeysClass.new).sort.should == ['a', 'b', 'c']
   end
+  
+  describe "with internal set" do
+    
+    it "returns instance variables of something that isn't a hash" do
```

Comment:
`for a Hash`

Reply:
I don't know what you mean, sorry?

Reply:
The spec description is not matching the spec (and duplicate of the spec description below): this example uses a Hash.

Reply:
Fixed.